unit tests failing on mac

### Is there an existing issue for this?

- [x] I confirm that I have not found an existing or similar issue.

### Feature

Failures: [m
[ERROR] [1;31m  ChromaVectorDatabaseEngineUnitTests.testListDocuments:479 expected: <newFile1.txt> but was: <newFile4.txt>[m
[ERROR] [1;31m  WeaviateVectorDatabaseEngineUnitTests.testListDocuments:683 expected: <newFile1.txt> but was: <newFile4.txt>[m

### Component / Function

_No response_

### Given

_No response_

### When

_No response_

### Then

_No response_

--- a/test/prerna/engine/impl/vector/ChromaVectorDatabaseEngineUnitTests.java
+++ b/test/prerna/engine/impl/vector/ChromaVectorDatabaseEngineUnitTests.java
@@ -476,7 +476,7 @@ public class ChromaVectorDatabaseEngineUnitTests {
 				assertEquals(fileNames.size(), docsOutput.size());
 				for (int fileIdx = 0; fileIdx < fileNames.size(); fileIdx++) {
 					Map<String, Object> outputDoc = docsOutput.get(fileIdx); 
-					assertEquals(fileNames.get(fileIdx), outputDoc.get("fileName"));
+					assertTrue(fileNames.contains(outputDoc.get("fileName")));
 					assertEquals(0.0, outputDoc.get("fileSize"));
 					LocalDateTime fileDateTime = LocalDateTime.parse((String) outputDoc.get("lastModified"),
 							DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss"));

--- a/test/prerna/engine/impl/vector/WeaviateVectorDatabaseEngineUnitTests.java
+++ b/test/prerna/engine/impl/vector/WeaviateVectorDatabaseEngineUnitTests.java
@@ -680,7 +680,7 @@ public class WeaviateVectorDatabaseEngineUnitTests {
 				assertEquals(fileNames.size(), docsOutput.size());
 				for (int fileIdx = 0; fileIdx < fileNames.size(); fileIdx++) {
 					Map<String, Object> outputDoc = docsOutput.get(fileIdx); 
-					assertEquals(fileNames.get(fileIdx), outputDoc.get("fileName"));
+					assertTrue(fileNames.contains(outputDoc.get("fileName")));
 					assertEquals(0.0, outputDoc.get("fileSize"));
 					LocalDateTime fileDateTime = LocalDateTime.parse((String) outputDoc.get("lastModified"),
 							DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss"));

